Source/Core/VideoBackends/OGL/Render.cpp

```diff
@@ -644,6 +646,8 @@ Renderer::Renderer()
 	glDisable(GL_STENCIL_TEST);
 	glEnable(GL_SCISSOR_TEST);
 
+	if (g_ActiveConfig.backend_info.bSupportsClipControl)
+		glClipControl(GL_LOWER_LEFT, GL_ZERO_TO_ONE);
```

Comment:
It will ofcourse affect it in accuract, but it doesn't need and code changes since we already make sure our depth buffer only uses the 0:1 range.